runner/proto/runner.proto

```
@@ -13,6 +17,7 @@ enum ExecutionStatus {
 message ConfigureRequest {
   map<string, string> config = 1;
   google.protobuf.Struct policy_data = 2;
+  map<string, StringList> policyBehavior = 3;
```

Buf (1.69.0)

[error] Field name "policyBehavior" should be lower_snake_case, such as "policy_behavior".

(FIELD_LOWER_SNAKE_CASE)